docs/sources/flow/config-language/expressions/types_and_values.md

```diff
@@ -88,6 +88,31 @@ The supported escape sequences are as follows:
 | `\uNNNN` | A Unicode character from the basic multilingual plane (NNNN is four hexadecimal digits) |
 | `\UNNNNNNNN` | A Unicode character from supplementary planes (NNNNNNNN is eight hexadecimal digits) |
 
+## Raw strings
+
+Raw strings are represented by sequences of Unicode characters surrounded by backticks ``` `` ```. 
+Raw strings do not support any escape sequences:
+
+```river
+`Hello, "world"!`
+```
+
+Within the backticks, any character may appear except a backtick.
```

Comment:
Do we want to mention that if someone _does_ want to insert a backtick, they can do so using string concatenation?